kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py

```diff
@@ -395,6 +395,10 @@ def build_entity(cls, record, get_target_id_from_record):
             "method_of_sample_procurement": record.get(
                 CONCEPT.BIOSPECIMEN.SAMPLE_PROCUREMENT
             ),
+            "dbgap_consent_code": record.get(
+                CONCEPT.BIOSPECIMEN.DBGAP_CONSENT_CODE
```

Comment:
It still uses `DBGAP_CONSENT_CODE`, not `CONSENT_CODE`

Reply:
correct - I haven't made the change yet. wasn't clear if we wanted that change

Reply:
```suggestion
                CONCEPT.BIOSPECIMEN.CONSENT_CODE
```